lib/graphql/lang/ast_reducer.ex

```diff
@@ -0,0 +1,105 @@
+
+# Used for computing a result as a function of the an AST traversal.  The
```

Comment:
I'm curious, I didn't find anything named _visitor_ when [searching in the elixir code](https://github.com/elixir-lang/elixir/search?utf8=%E2%9C%93&q=visitor&type=Code) so I wonder how they solved this problem.